WordPress/gutenberg
# Resizable Editor: Make the editor resizable with arrow keys

## What?

I found that although the resize handles of the resizable editor are keyboard focusable, pressing the left or right key takes the focus away, making it impossible to resize the editor.

## Why?

This is probably due to the influence of other event handlers.

## How?

As with [here](https://github.com/WordPress/gutenberg/blob/da16125f6f6d7099207fc3d5d2da2ed8c2927022/packages/edit-site/src/components/resizable-frame/index.js#L171), check the key and add `event.preventDefault()`.

## Testing Instructions

- Open the pattern editor.
- Use the tab key to focus the drag handle.
- POresing the left and right key should resize the editor.


## Screenshots or screencast 

https://github.com/user-attachments/assets/aeca7c6e-76bd-49c8-8a18-8a100609c9bd
